### PR TITLE
Add Android Q compiling support

### PR DESCRIPTION
**Pull request by LukeZheZhu**

### CODE REVIEW

File: omx_buffers/Android.mk [hunk 1]
```diff
@@ -13,7 +13,8 @@ LOCAL_C_INCLUDES := \
 
 LOCAL_CFLAGS := \
     $(MFX_OMX_CFLAGS) \
-    $(MFX_OMX_CFLAGS_LIBVA)
+    $(MFX_OMX_CFLAGS_LIBVA) \
+	-Wno-error
```
Comment (dmitrybrazhkin) [inline omx_buffers/Android.mk]: Not good idea to ignore all warnings

File: omx_components/Android.mk [hunk 2]
```diff
@@ -15,7 +15,8 @@ LOCAL_C_INCLUDES := \
 
 LOCAL_CFLAGS := \
     $(MFX_OMX_CFLAGS) \
-    $(MFX_OMX_CFLAGS_LIBVA)
+    $(MFX_OMX_CFLAGS_LIBVA) \
+	-Wno-error
```
Comment (dmitrybrazhkin) [inline omx_components/Android.mk]: Not good idea to ignore all warnings

File: omx_utils/Android.mk [hunk 3]
```diff
@@ -10,11 +10,13 @@ LOCAL_SRC_FILES := \
...
     $(MFX_OMX_INCLUDES_LIBVA) \
-    external/tinyxml2
+    external/tinyxml2 \
+	$(MFX_OMX_HOME)/../../../../../frameworks//native/headers/media_plugin/media/openmax/
 
 LOCAL_CFLAGS := \
     $(MFX_OMX_CFLAGS) \
-    $(MFX_OMX_CFLAGS_LIBVA)
+    $(MFX_OMX_CFLAGS_LIBVA) \
+    -Wno-error
```
Comment (dmitrybrazhkin) [inline omx_utils/Android.mk]: Not good idea to ignore all warnings

File: omx_buffers/include/mfx_omx_bst_ibuf.h [hunk 4]
```diff
@@ -1,4 +1,5 @@
-// Copyright (c) 2011-2019 Intel Corporation
+
+// Copyright (c) 2011-2018 Intel Corporation
```
Comment (dmitrybrazhkin) [inline omx_buffers/include/mfx_omx_bst_ibuf.h]: Please revert

File: omx_utils/Android.mk [hunk 5]
```diff
@@ -10,11 +10,13 @@ LOCAL_SRC_FILES := \
 LOCAL_C_INCLUDES := \
     $(MFX_OMX_INCLUDES) \
     $(MFX_OMX_INCLUDES_LIBVA) \
-    external/tinyxml2
+    external/tinyxml2 \
+	$(MFX_OMX_HOME)/../../../../../frameworks//native/headers/media_plugin/media/openmax/
```
Comment (dmitrybrazhkin) [inline omx_utils/Android.mk]: Please remove that. OMX headers are included through media_plugin_headers in LOCAL_HEADER_LIBRARIES

Comment (LukeZheZhu) [reply omx_utils/Android.mk]: Hi @dmitryermilov ，
I have updated my PR. Please help to review. Thank you so much.

File: mfx_omx_defs.mk [hunk 6]
```diff
@@ -48,7 +51,7 @@ MFX_OMX_CFLAGS += \
 
 ifeq ($(BOARD_USES_GRALLOC1),true)
   MFX_OMX_CFLAGS += -DMFX_OMX_USE_GRALLOC_1
-  ifneq ($(filter MFX_P ,$(MFX_ANDROID_VERSION)),)
+	ifneq ($(filter MFX_P MFX_Q,$(MFX_ANDROID_VERSION)),)
```
Comment (dmitrybrazhkin) [inline mfx_omx_defs.mk]: Tabs are not allowed in MSDK/OMX code

Comment (TianmiChen) [reply mfx_omx_defs.mk]: @LukeZheZhu , please remove the tabs here.

Thanks,

**Pull request merged**